core/package.json

```diff
@@ -13,18 +13,18 @@
   "dependencies": {
     "@redmedical/hvstr-utils": "1.0.0",
     "mkdirp": "^0.5.1",
-    "protractor": "~5.1.2",
+    "protractor": "^5.4.2",
     "require-from-string": "^2.0.2",
-    "ts-node": "^7.0.1"
+    "ts-node": "^8.1.0"
   },
   "devDependencies": {
-    "@types/jasmine": "^3.3.12",
-    "@types/node": "^10.9.4",
-    "jasmine": "^3.3.1",
-    "nyc": "^13.3.0",
-    "tslint": "~5.9.1",
+    "@types/jasmine": "^3.3.13",
+    "@types/node": "^10.14.10",
+    "jasmine": "^3.4.0",
+    "nyc": "^14.1.1",
+    "tslint": "^5.16.0",
     "typedoc": "^0.14.0",
-    "typescript": "^3.0.3"
+    "typescript": "^3.4.5"
```

Comment:
I would prevent using different versions of typescript or any other dependency in the same project.